lib/deploy/functions/prepare.js

```diff
@@ -58,9 +58,10 @@ module.exports = function(context, options, payload) {
   var projectId = getProjectId(options);
   var instance = options.instance;
   return RSVP.all([
-    ensureApiEnabled(options.project, 'cloudfunctions.googleapis.com', 'functions'),
-    ensureApiEnabled(projectId, 'runtimeconfig.googleapis.com', 'runtimeconfig')
-  ]).then(function() {
+    ensureApiEnabled.ensure(options.project, 'cloudfunctions.googleapis.com', 'functions'),
+    ensureApiEnabled.check(projectId, 'runtimeconfig.googleapis.com', 'runtimeconfig', true)
+  ]).then(function(results) {
+    _.set(context, 'configEnabled', results[1]);
```

Comment:
`runtimeConfigEnabled` -- we have too many things called "config" to have this be clear on its own